armi/reactor/blocks.py

```diff
@@ -1529,6 +1530,20 @@ def rotate(self, deg):
         """
         raise NotImplementedError
 
+    def setTargetComponent(self, targetComponent):
```

Comment:
suggest rename. target component could be different for different types of physics. for depletion target component would be fuel, for assembly distortion in might be some structure. Thermal expansion should not reserve the total namespace of  `setTargetComponent`

Something like `setThermalExpComp` would be better

Reply:
How about `setThermalExpTargetComp`? In the interest of being consistent with the "target component" in the axial expansion changer.

Reply:
In addition to changing the name of this method, it might also be useful to change the name of the `self.targetComponent` attribute here and in `blockBlueprint.py`.

Reply:
Addressed https://github.com/terrapower/armi/pull/777/commits/293a0ef98b64bb85055814dbbe3923d955123c42